static/settings.js

```diff
@@ -118,6 +128,13 @@ define(function (require) {
         });
         add(root.find('.hoverShowSource'), 'hoverShowSource', true, Checkbox);
 
+        var formats = ["Google", "LLVM", "Mozilla", "Chromium", "WebKit", "None"];
```

Comment:
Temporal way of doing this. Setings releated to formatting should have explicit language/program name in its name so later on supporting new languages is not as hard